src/pudl/package_data/eia860/column_maps/generator_wind_existing.csv

```diff
@@ -0,0 +1,25 @@
+plant_id_eia,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code
+plant_name_eia,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name
+state,state,state,state,state,state,state,state,state,state,state
+county,county,county,county,county,county,county,county,county,county,county
+generator_id,generator_id,generator_id,generator_id,generator_id,generator_id,generator_id,generator_id,generator_id,generator_id,generator_id
+operational_status_code,status,status,status,status,status,status,status,status,status,status
+technology_description,,technology,technology,technology,technology,technology,technology,technology,technology,technology
+prime_mover_code,prime_mover,prime_mover,prime_mover,prime_mover,prime_mover,prime_mover,prime_mover,prime_mover,prime_mover,prime_mover
+sector_name_eia,sector_name,sector_name,sector_name,sector_name,sector_name,sector_name,sector_name,sector_name,sector_name,sector_name
+sector_id_eia,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector
+capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw
+summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw
+winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw
+generator_operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month
+generator_operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year
+predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer
+predominant_turbine_model_num,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number
```

Comment:
I think the `_num` suffix here would be interpreted as "number of different turbine models" but I suspect that what's really being reported here is like "GE Haliade-X14" or something identifying the type of turbines installed. In which case maybe this should be `predominant_turbine_model` (or `_model_id` or `_model_type`)?

Reply:
good point! i like `_model_id`!